randovania/games/am2r/json_data/Industrial Complex.txt

```diff
@@ -266,20 +270,20 @@ Extra - map_name: rm_a3h06
   * Layers: default
   * Open Passage to Exterior Gamma Nest Access/Dock to Exterior Gamma Nest
   > Top
-      Space Jump Wall
+      Power Grip Climb
```

Comment:
cant this be done with high mid air morph climb?

Reply:
Isn't `power grip climb` the lesser template?

Reply:
no
power grip climb forbids any spring combination. High mid-air morph allows you to get in with bombs+spring, or HJ+spring

(could've easily answered that by just looking at the templates btw :P)